src/applications/disability-benefits/all-claims/pages/unemployabilityFormUpload.js

```diff
@@ -0,0 +1,61 @@
+import fileUploadUI from 'us-forms-system/lib/js/definitions/file';
+import environment from '../../../../platform/utilities/environment';
+import { FIFTY_MB } from '../constants';
+import { DocumentDescription } from '../content/uploadFormDocuments';
+
+export const uiSchema = {
+  'ui:title': 'Upload VA Form 21-8940',
```

Comment:
So, I don't think you need to specify a `ui:title` or `ui:description` for this page's schema. What you might want to try is `form8940Upload: fileUploadUI('Upload VA Form 21-8940'...` and then merge the `fileUploadUI` with `{ 'ui:description': DocumentDescription }`. For an example, you can check out the `additionalDocuments` uiSchema